sdk/examples/intkey_jvm_sc/sawtooth_intkey/cli/load.py

```diff
@@ -41,30 +42,20 @@ def _split_batch_list(batch_list):
 
 def do_load(args):
     print("Do load")
-    with open(args.filename, 'rb') as fd:
+    with open(args.filename, mode='rb') as fd:
         batches = batch_pb2.BatchList()
         batches.ParseFromString(fd.read())
 
-    stream = Stream(args.url)
-    futures = []
     start = time.time()
 
     for batch_list in _split_batch_list(batches):
-        future = stream.send(
-            message_type=Message.CLIENT_BATCH_SUBMIT_REQUEST,
-            content=batch_list.SerializeToString())
-        futures.append(future)
-
-    for future in futures:
-        result = future.result()
-        assert(result.message_type == Message.CLIENT_BATCH_SUBMIT_RESPONSE)
+        post_batches(args.url, batch_list)
```

Comment:
Futures is a better API, because the SDK client might want to do other things while batches are being posted, like printing out status.

This also requires post_batches() to finish before going on to the next one, which worse than the code it replaces which allowed multiple batch lists to be sent before the previous had completely finished.